docs/en/02_Developer_Guides/14_Files/02_Images.md

```diff
@@ -283,7 +282,7 @@ class Page extends SiteTree
 }
 ```
 
-#### Controlling lazy loading in for `<img>` tags in SS templates
+#### Controlling lazy loading in for <img> tags in SS templates
```

Comment:
That looks more like a bug with our MD parser for the dev site. I would rather we fix it there if it's possible.

If that's not trivial, I'm happy to merge this specific change now since this page is about to get a lot of attention

Reply:
I'd say non-trivial?